kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -355,35 +357,31 @@ public static CompoundStat percentiles(Map<MetricName, Double> percentiles) {
     }
   }
 
-  public static class MetricsBuilder {
+  private static final class MetricsBuilder {
 
-    private MetricsContext metricsContext;
-    private JmxReporter reporter;
+    final MetricsContext metricsContext;
+    final JmxReporter reporter;
     // 1 sample per second
-    private int numSamples = 10;
-    private long sampleWindowMs = 10000;
-    private Time time = Time.SYSTEM;
+    final int numSamples = 10;
+    final long sampleWindowMs = 10000;
```

Comment:
I'm not sure these make sense. If we are always going to use these values for them, let's make them... - you guessed it, compile time constants :)

Reply:
Done.